Mathlib/CategoryTheory/Triangulated/Functor.lean

```diff
@@ -96,6 +96,8 @@ set_option maxHeartbeats 400000 in
 noncomputable instance [∀ (n : ℤ), (shiftFunctor C n).Additive]
     [∀ (n : ℤ), (shiftFunctor D n).Additive] : (F.mapTriangle).CommShift ℤ where
   iso := F.mapTriangleCommShiftIso
+  zero := by ext <;> simp
+  add _ _ := by ext <;> simp
```

Comment:
Maybe a comment on these: "-- these work by the `auto_param` but are even slower"